packages/react-inbox/src/components/Inbox/LazyTippy.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef, useState } from "react";
+import Tippy, { TippyProps } from "@tippyjs/react";
+
+const LazyTippy: React.FunctionComponent<TippyProps> = forwardRef(
+  (props, ref) => {
+    const [mounted, setMounted] = useState(false);
+
+    const lazyPlugin = {
+      fn: () => ({
```

Comment:
Could we add a comment here what `fn` means? or how it will be used

Reply:
ill add a link to the gist i stole it from, i think its part of how you write tippy plugins